### PR TITLE
Capacity tooltip

### PR DESCRIPTION
**Pull request by Camwyn**
Ticket: https://central.tri.be/issues/80616

Add the styles here - since we're already styling it here. Should explore moving it to ET+ though.

Structure in https://github.com/moderntribe/event-tickets-plus/pull/353

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:74](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/edit_admin_panel.php#L74)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startHourOptions' | [edit_admin_panel.php:76](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/edit_admin_panel.php#L76)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMinuteOptions' | [edit_admin_panel.php:79](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/edit_admin_panel.php#L79)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMeridianOptions' | [edit_admin_panel.php:83](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/edit_admin_panel.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:95](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/edit_admin_panel.php#L95)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endHourOptions' | [edit_admin_panel.php:97](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/edit_admin_panel.php#L97)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMinuteOptions' | [edit_admin_panel.php:100](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/edit_admin_panel.php#L100)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMeridianOptions' | [edit_admin_panel.php:104](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/edit_admin_panel.php#L104)
:x: | empty() and isset() must only contain variables (PHP 5.2 compatibility) | [list.php:60](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/list.php#L60)
:x: | There must be a single space after the first parenthesis | [list.php:60](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/list.php#L60)
:x: | There must be a single space before the last parenthesis | [list.php:60](https://github.com/moderntribe/event-tickets/blob/f96d820b0e4337622d22be54e99b3b39c98a866e/src/admin-views/list.php#L60)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:74](https://github.com/moderntribe/event-tickets/blob/80f3b30be9cb938a26c4558f9967af5e85b162a2/src/admin-views/edit_admin_panel.php#L74)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startHourOptions' | [edit_admin_panel.php:76](https://github.com/moderntribe/event-tickets/blob/80f3b30be9cb938a26c4558f9967af5e85b162a2/src/admin-views/edit_admin_panel.php#L76)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMinuteOptions' | [edit_admin_panel.php:79](https://github.com/moderntribe/event-tickets/blob/80f3b30be9cb938a26c4558f9967af5e85b162a2/src/admin-views/edit_admin_panel.php#L79)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMeridianOptions' | [edit_admin_panel.php:83](https://github.com/moderntribe/event-tickets/blob/80f3b30be9cb938a26c4558f9967af5e85b162a2/src/admin-views/edit_admin_panel.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:95](https://github.com/moderntribe/event-tickets/blob/80f3b30be9cb938a26c4558f9967af5e85b162a2/src/admin-views/edit_admin_panel.php#L95)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endHourOptions' | [edit_admin_panel.php:97](https://github.com/moderntribe/event-tickets/blob/80f3b30be9cb938a26c4558f9967af5e85b162a2/src/admin-views/edit_admin_panel.php#L97)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMinuteOptions' | [edit_admin_panel.php:100](https://github.com/moderntribe/event-tickets/blob/80f3b30be9cb938a26c4558f9967af5e85b162a2/src/admin-views/edit_admin_panel.php#L100)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMeridianOptions' | [edit_admin_panel.php:104](https://github.com/moderntribe/event-tickets/blob/80f3b30be9cb938a26c4558f9967af5e85b162a2/src/admin-views/edit_admin_panel.php#L104)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Ticket_Object.php [hunk 1]
```diff
@@ -407,6 +407,26 @@ public function display_original_stock() {
 			return $orginal_stock;
 		}
 
+
+		/**
+		 * Method to display the available `stock` propriety of the Object.
+		 *
+		 * Returns the current ticket available capacity: either an integer or a
+		 * string if stock is unlimited.
+		 *
```
Comment (bordoni) [inline src/Tribe/Ticket_Object.php]: Add `@since`

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:74](https://github.com/moderntribe/event-tickets/blob/cbb5f58b6498f19d349723f7298f90c3189d8ce0/src/admin-views/edit_admin_panel.php#L74)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startHourOptions' | [edit_admin_panel.php:76](https://github.com/moderntribe/event-tickets/blob/cbb5f58b6498f19d349723f7298f90c3189d8ce0/src/admin-views/edit_admin_panel.php#L76)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMinuteOptions' | [edit_admin_panel.php:79](https://github.com/moderntribe/event-tickets/blob/cbb5f58b6498f19d349723f7298f90c3189d8ce0/src/admin-views/edit_admin_panel.php#L79)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMeridianOptions' | [edit_admin_panel.php:83](https://github.com/moderntribe/event-tickets/blob/cbb5f58b6498f19d349723f7298f90c3189d8ce0/src/admin-views/edit_admin_panel.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:95](https://github.com/moderntribe/event-tickets/blob/cbb5f58b6498f19d349723f7298f90c3189d8ce0/src/admin-views/edit_admin_panel.php#L95)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endHourOptions' | [edit_admin_panel.php:97](https://github.com/moderntribe/event-tickets/blob/cbb5f58b6498f19d349723f7298f90c3189d8ce0/src/admin-views/edit_admin_panel.php#L97)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMinuteOptions' | [edit_admin_panel.php:100](https://github.com/moderntribe/event-tickets/blob/cbb5f58b6498f19d349723f7298f90c3189d8ce0/src/admin-views/edit_admin_panel.php#L100)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMeridianOptions' | [edit_admin_panel.php:104](https://github.com/moderntribe/event-tickets/blob/cbb5f58b6498f19d349723f7298f90c3189d8ce0/src/admin-views/edit_admin_panel.php#L104)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Ticket_Object.php (hunk 1 above)
Comment (Camwyn) [inline src/Tribe/Ticket_Object.php]: That function was created but isn't used - removing.

File: src/admin-views/edit_admin_panel.php [hunk 2]
```diff
@@ -16,16 +16,17 @@
...
 				<div class="input_block">
-					<label class="ticket_form_label" for="ticket_name"><?php esc_html_e( 'Type:', 'event-tickets' ); ?></label>
-					<input type='text' id='ticket_name' name='ticket_name' class="ticket_field" size='25' value='' />
+					<label class="ticket_form_label ticket_form_left" for="ticket_name"><?php esc_html_e( 'Type:', 'event-tickets' ); ?></label>
+					<input type='text' id='ticket_name' name='ticket_name' class="ticket_field ticket_form_right" size='25' value='' />
+					<span class="tribe_soft_note ticket_form_right"><?php esc_html_e( 'Ticket type name shows on the front end and emailed tickets', 'event-tickets' ); ?></span>
 				</div>
 				<fieldset class="input_block">
-					<legend class="ticket_form_label"><?php esc_html_e( 'Sell using:', 'event-tickets' ); ?></legend>
+					<legend class="ticket_form_label"><?php esc_html_e( 'Sell using (visible for testing):', 'event-tickets' ); ?></legend>
```
Comment (borkweb) [inline src/admin-views/edit_admin_panel.php]: What is ` (visible for testing)` mean? Is that in there on purpose?

Comment (Camwyn) [inline src/admin-views/edit_admin_panel.php]: Yes - hidden when the capacity functionality is all done - allow us to test switching and the different UI for each payment method.

**Comment by borkweb** [comment]
:+1:

**Pull request merged**